package.json

```diff
@@ -28,6 +28,7 @@
     "@types/react-dom": "17.0.3",
     "@types/socket.io-client": "1.4.36",
     "browser-fs-access": "0.18.0",
+    "canvas": "2.8.0",
```

Comment:
looks like this was pushed by mistake, can you remove it ?

Reply:
done